### PR TITLE
extend original-data in id3

### PR DESCRIPTION
**Pull request by grmkris**
**JIRA task url** (if applicable)

**(If Bug) What was the problem being solved?**

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

**How to reproduce fix / test this PR?**

### CODE REVIEW

File: polygon-id/id3/id3.json [hunk 1]
```diff
@@ -1804,8 +1804,609 @@
               "type": "object",
               "properties": {
                 "citizenship": {
-                  "title": "Citizenship",
-                  "description": "The country of citizenship of the credential subject - ISO 3166-1 alpha-3 country code (https://www.iban.com/country-codes)",
+                  "title": "Citizenship - ISO 3166-1 alpha-3 country code (https://www.iban.com/country-codes)",
```
Comment (xela7) [inline polygon-id/id3/id3.json:1807]: do you really want the title to be the long title instead of Citizenship?

**Comment by xela7** [comment]
one note:

if you import the schema into the schema builder, it turns certain atttributes red when it doesn't think that they are valid.

The schema builder wants to see all attributes having a title and a description.  I don't think this is strictly necessary in json schemas.  

One such location that i was surprised by is on ammendments.data:

https://github.com/NexeraProtocol/vc-schemas/blob/40138e2f738845c50d04a09b8fd690f70a139069/polygon-id/id3/id3.json#L43C20-L43C20

so i don't know if we want to define this or not -- but container attributes appear to be defineable with title/description as well as nested.

**Comment by xela7** [comment]
i have a few changes on a separate branch that might be beneficial leftover from last week -- 

https://github.com/NexeraProtocol/vc-schemas/compare/id3-improvements...possible-title-description-updates

**Comment by xela7** [comment]
citizenship title/description change could happen in 4 places:

https://github.com/NexeraProtocol/vc-schemas/blob/a4a6524beda493e753fa6c8cade03dd288969868/polygon-id/id3/id3.json#L46

https://github.com/NexeraProtocol/vc-schemas/blob/a4a6524beda493e753fa6c8cade03dd288969868/polygon-id/id3/id3.json#L1807

https://github.com/NexeraProtocol/vc-schemas/blob/a4a6524beda493e753fa6c8cade03dd288969868/polygon-id/id3/id3.json#L3453

File: polygon-id/id3/id3.json (hunk 1 above)
Comment (grmkris) [reply polygon-id/id3/id3.json:1829]: let's fix it

**Comment by grmkris** [comment]
- let's add title and descriptions where missing
- let's fix citizenship 

https://github.com/NexeraProtocol/vc-schemas/pull/92#issuecomment-1775581140 about this one i am not sure what you mean, i don't see it on url, can you screenshot it

File: polygon-id/id3/id3.json (hunk 1 above)
Comment (xela7) [reply polygon-id/id3/id3.json]: fixed

**Comment by xela7** [comment]
Ok all changes have been made.

no need to worry about https://github.com/NexeraProtocol/vc-schemas/pull/92#issuecomment-1775581140.  These changes were just title/description changes as well.

**Comment by grmkris** [comment]
please approve and we merge this

**Review by xela7 — APPROVED** [review]

**Pull request merged**